tsp_simple.cc

```diff
@@ -135,8 +137,16 @@ void MissionsBuilder(const TSPTWDataDT& data, RoutingModel& routing,
         }
         if (due.back() < CUSTOM_MAX_INT) {
           if (late_multiplier > 0) {
-            routing.GetMutableDimension(kTime)->SetCumulVarSoftUpperBound(
-                index, due.back(), late_multiplier);
+            if (initial_value >= 0) {
+              assignment->Add(cumul_var);
+              if (initial_value > due.back())
+                assignment->SetMax(cumul_var, initial_value);
+              else
+                assignment->SetMax(cumul_var, due.back());
+            } else {
+              routing.GetMutableDimension(kTime)->SetCumulVarSoftUpperBound(
+                  index, due.back(), late_multiplier);
```

Comment:
If we don't add this cost, when we launch the optimisation second time (to check if we can move the pause or not) the cost will be smaller then the original :/ we have to keep the cost, right?

Reply:
correct !